src/OpenApi/src/Services/Schemas/OpenApiSchemaService.cs

```diff
@@ -71,7 +76,7 @@ internal async Task<OpenApiSchema> GetOrCreateSchemaAsync(Type type, ApiParamete
         var schemaAsJsonObject = _schemaStore.GetOrAdd(key, CreateSchema);
         if (parameterDescription is not null)
         {
-            schemaAsJsonObject.ApplyParameterInfo(parameterDescription);
+            schemaAsJsonObject.ApplyParameterInfo(parameterDescription, OpenApiJsonSchemaContext.Default.GetTypeInfo(type)!);
```

Comment:
Could you explain why this is always non-null?

Reply:
🤦🏽‍♀️ Good catch -- goof on my part.